test/integration/fleetscope/fleetscope_test.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func TestFleetscope(t *testing.T) {
-
+	setupOutput := tft.NewTFBlueprintTest(t, tft.WithSetupPath("../../setup"), tft.WithTFDir("../../../"))
```

Comment:
this needs to change to retrieve an object

Reply:
Got the following error in the CI log:

Error: Output "teams" not found
        	            	
        	            	The output variable requested could not be found in the state file.

Reply:
Oh - I see you added it to the test setup, let me take a closer look at your code and the CI output.

Reply:
Ok, here it is in the setup ouput:

```
Step #0 - "prepare": teams = {
Step #0 - "prepare":   "accounts-team" = "accounts-team-8xant6@test.blueprints.joonix.net"
Step #0 - "prepare":   "frontend-team" = "frontend-team-8xant6@test.blueprints.joonix.net"
Step #0 - "prepare":   "transactions-team" = "transactions-team-8xant6@test.blueprints.joonix.net"
Step #0 - "prepare": }
```

Reply:
```
Step #1 - "bootstrap-init": TestBootstrap 2024-06-05T03:30:45Z terraform.go:294: output key "teams": map[accounts-team:accounts-team-8xant6@test.blueprints.joonix.net frontend-team:frontend-team-8xant6@test.blueprints.joonix.net transactions-team:transactions-team-8xant6@test.blueprints.joonix.net]
```

Reply:
Still looks good in fleetscope-init

```
TestFleetscope 2024-06-05T03:47:25Z terraform.go:294: output key "teams": map[accounts-team:accounts-team-8xant6@test.blueprints.joonix.net frontend-team:frontend-team-8xant6@test.blueprints.joonix.net transactions-team:transactions-team-8xant6@test.blueprints.joonix.net]
```

Reply:
Here is the problem, that's not the right path:

```
TestFleetscope 2024-06-05T03:47:26Z terraform.go:264: Running tests TF configs in ../../../ with version 1.8.4
TestFleetscope 2024-06-05T03:47:26Z retry.go:91: terraform [output -no-color -json teams]
TestFleetscope 2024-06-05T03:47:26Z command.go:100: Running command terraform with args [output -no-color -json teams]
TestFleetscope 2024-06-05T03:47:26Z command.go:185: 
TestFleetscope 2024-06-05T03:47:26Z command.go:185: Error: Output "teams" not found
```

Reply:
Thanks @apeabody , fixed the path, see if it works ...

Reply:
@yliaog - Rather than creating a separate `setup := tft.NewTFBlueprintTest`, as the `teams` object is created during the setup phase and stored in the gcs backend.  I would suggest trying to use one of the later existing `NewTFBlueprintTest` which is already configured to use `tft.WithBackendConfig(backendConfig)` and should include the desired setup output.

Reply:
i see. changed the code as you suggested, see how it works.

Reply:
Oh! as `teams` was created in setup rather than a "regular" stage, we'll need to use a `GetTFSetupOutput()` to receive rather than an `Output()` function.  Forgot about that when I wrote the above.  Unfortunately we currently only have `GetTFSetupOutput` for strings and lists, not maps.

Ref: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/blueprint-test/pkg/tft/terraform.go#L388

While we get that added, your plan to use something like this, is probably the most quickest option:

```
	setup := tft.NewTFBlueprintTest(t,
		tft.WithTFDir("../../setup"),
	)

	"teams": testutils.GetBptOutputStrSlice(setup, "teams")
```